code/game/jobs/job/prospector.dm

```diff
@@ -105,6 +106,7 @@
 	spawn_positions = 2
 	supervisors = "the Foreman"
 	difficulty = "Medium."
+	alt_titles = list("Pathfinder","Sapper","Scout","Frontiersmen")
```

Comment:
Not sure if pathfinder is a good idea

Reply:
Why not?

Reply:
I dont think they are explorers or really SI related

Reply:
Why so?

Reply:
I  mean, prospectors open the way for the salvagers, they are frontliners

Reply:
Prospectors are the muscles, Salvagers are the brains, Foreman's supposed to be both.